appbanners.links.menu.yml

```diff
@@ -0,0 +1,5 @@
+appbanners.settings:
+  title: 'App Banners'
+  parent: system.admin_config
+  route_name: appbanners.admin_settings
+  
```

Comment:
the file shouldn't have trailing whitespace - please could you run this through PHP Code Sniffer if you can?

see https://www.drupal.org/node/1419980 and https://www.drupal.org/docs/develop/standards/coding-standards